packages/button/src/Button.spec.tsx

```diff
@@ -138,4 +138,16 @@ describe('packages/Button', () => {
 
     expect(buttonComponent.tagName.toLowerCase()).toBe('div');
   });
+
+  test(`renders a button with an arbitrary element passed in glyph prop`, () => {
+    const buttonComponent = render(
+      <Button glyph={<svg />}>My Trash Button</Button>,
+    ).container.firstChild;
+
+    if (!typeIs.element(buttonComponent)) {
+      throw new Error('No element was rendered');
+    }
+
+    expect(buttonComponent.getAttribute('glyph')).not.toBe(null);
```

Comment:
Right now this test is testing if the buttonComponent as rendered in the HTML contains a glyph attribute, which is currently occurring but is not the behavior that we want. I'd suggest that first you make sure that we're not spreading the glyph prop along with {...rest} on the Button component and make sure that this test fails. Then, add another test (perhaps checking that the buttonComponent.innerHTML contains <svg>, but feel free to get more creative than that) to ensure that you're testing that we are rendering the desired svg.

Reply:
Ok, I went ahead and modified the component and the test to account for this. I'm now checking if the component's innerHTML contains "svg" instead.

Reply:
:+1